### PR TITLE
Fixed issue-content shifting when skip to content

### PR DESCRIPTION
**Pull request by phudson-he**
Changed the way the "Skip to content" link works upon initial keyboard tab press

### CODE REVIEW

File: arches/app/templates/base-manager.htm [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
     <div class="wrapper">
 
-        <a id="topskip" tabindex="1" href="#main-content">Skip to content</a>
+        <div id="skip-link-holder"><a id="skip-link" href="#skiptocontent">Skip to content</a></div>
```
Comment (aj-he) [inline arches/app/templates/base-manager.htm]: does this need a tabindex?

**Review by aj-he — COMMENTED** [review]
just the one comment

File: arches/app/templates/base-manager.htm (hunk 1 above)
Comment (phudson-he) [reply arches/app/templates/base-manager.htm]: Didn't need it, but added it there just in case some tabbable element gets added before it in the future.

**Review by aj-he — APPROVED** [review]
looks good

**Pull request merged**